CondCore/Utilities/src/CondDBFetch.cc

```diff
@@ -56,7 +56,6 @@ namespace cond {
       FETCH_PAYLOAD_CASE(CTPPSPixelAnalysisMask)
       FETCH_PAYLOAD_CASE(CTPPSPixelGainCalibrations)
       FETCH_PAYLOAD_CASE(CTPPSRPAlignmentCorrectionsData)
```

Comment:
@ggovi is this deletion fine?

Reply:
@tvami It is only relevant for conddb_import, which is an obsolete command anyhow... Therefore yes, it is fine.